### ISSUE TITLE
Feature Suggestions

### ISSUE THREAD

**Issue by kiehtan** [peterhajas/MobileNotifier#105]
1. Notifications that rotate into landscape mode.
2. SB Rotator4 support for messages stored in the multi tasking section.
3. Setting option to set notification to appear at buttom of screen instead of top of screen. 
4. Setting option to set notifications to time out after so many seconds.
5. Ability to dismiss (x on right side like beta 1) or open (pressing the icon on the left side of the notification) notifications on the notification itself without having to tap open seperate open/later buttons.
6. Ability to repond to texts within the notification, similar to tlert.

**Comment by mergesort** [comment]
I agree with the points 1, 2, and 6 as mentioned above, especially point 6. 

I'd also like to be able to see the switcher dash while in an app. 
Another feature that is a necessity is scrolling text in the dash, or when you get a notification, so I can read through a whole notofication before deciding to take an action on it.

**Comment by btaroli** [comment]
AGREE: 1, 4
ABSTAIN: 2, 6
DISAGREE: 3 (breaks UI paradigm, selectable?), 5

Regarding #4, the "pop-up" already does time out after a brief period. If you meant make it customizable how long it appears, I'd agree.

Regarding #5, I think the goal is to make the pop-up as unobtrusive as possible. I rather liked beta 3's approach to swipe-to-dismiss (leave for later). If I wanted to take some other action, tapping is fine. Often, in situations where I shouldn't be reaching (like driving), I'm happy to just let the pop-up time out. I'm curious to see how it interacts with my GPS navigation app, but the proper pop-in auto-scaling of app window area seems to work thus far, so I'm not too worried.

Regarding #6, I like keeping MN as task-focused as possible without extra baggage. If such a feature was added, I'd want a way to turn it off.
